modules/nixos-nixpkgs-last-modified.nix

```diff
@@ -0,0 +1,3 @@
+{ inputs, ... }: {
+  environment.etc."nixpkgs-last-modified".text = toString inputs.nixpkgs.lastModified;
```

Comment:
This writes out the timestamp of the last git commit in nixpkgs used for building NixOS, so should be very accurate description of system's 'up-to-dateness".